mappings/net/minecraft/client/Minecraft.mapping

```diff
@@ -36,9 +38,11 @@ CLASS net/minecraft/client/Minecraft net/minecraft/client/Minecraft
 		ARG 0 playerName
 	METHOD b getGameDirectory ()Ljava/io/File;
 	METHOD c printOpenGLError (Ljava/lang/String;)V
+	METHOD d stop ()V
+	METHOD f scheduleStop ()V
 	METHOD g lockCursor ()V
 	METHOD j toggleFullscreen ()V
 	METHOD l hasLevel ()Z
 	METHOD m switchDimension ()V
-	METHOD u isFancyGraphicsEnabled ()Z
+	METHOD u isFancyGraphicsEnabled ()Z127
```

Comment:
```suggestion
	METHOD u isFancyGraphicsEnabled ()Z
```

Reply:
Not sure how this happened, only edited files through enigma. Bug?